### ISSUE TITLE
Inability to apply bindings

### ISSUE THREAD

**Issue by CitrusCell** [Unordinal/HollowKnight.ToggleableBindings#1]
I did not modify the configuration files at all. I was able to open the menu of bindings using "down+superdash" but failed to confirm. I pressed the apply button and it just got stuck there, making the game completely dead, and I had no choice but to press "Alt+F4" in order to shut the program down.
I cleared up all my other mods but the issue is still happening, or did I lack some of the components needed to run the mod?
![20210515083502_1](https://user-images.githubusercontent.com/84211393/118342964-83ec8f00-b558-11eb-835a-bb415c5df58c.jpg)
Thank you for your support.

**Comment by Unordinal** [comment]
Hey, thanks for the report! The mod _does_ require `Vasi`; make sure you have that installed. I don't believe I have that noted in the readme, so I'll make sure to include that somewhere.

If that doesn't fix it, could you cause the issue to happen again and then send me your `ModLog.txt` and `output_log.txt` files? They're located in `C:\Users\<your user name>\AppData\LocalLow\Team Cherry\Hollow Knight` and might help me diagnose the issue.

**Comment by CitrusCell** [comment]
Thanks for replying! Actually, I did install Vasi.dll before loading the game, but that didn't fix the problem.
The ModdingAPI I'm using is version 60, and the log files are enclosed below.
[ModLog.txt](https://github.com/Unordinal/HollowKnight.ToggleableBindings/files/6493491/ModLog.txt)
[output_log.txt](https://github.com/Unordinal/HollowKnight.ToggleableBindings/files/6493492/output_log.txt)

**Comment by Unordinal** [comment]
Thanks for the log files! Could you try [downloading this test version of ToggleableBindings](https://mega.nz/file/ZBdhGKDJ#b4s_rPqP174DYq6SzCgJabW6SmO0DzayzYbjiiIgV_k) and letting me know if it works for you?

It looks like your Vasi might not the exact one that's available through the mod installer as it's missing one of the methods I used in ToggleableBindings. Please let me know where you downloaded it from or if it actually _is_ from the installer.

**Comment by CitrusCell** [comment]
It seems that it has been my own problem all the time. ModInstaller is just unavailable in my country, so I had to install them manually all the time, which is the reason I was using the obsolete version of Vasi. I've tested your ToggleableBindings 0.1.3 using the outdated Vasi and it worked well!  
![20210518203235_1](https://user-images.githubusercontent.com/84211393/118655643-5172b880-b81c-11eb-9595-9fe3180c0b14.jpg)
I found a way to run ModInstaller eventually, your released 0.1.2 version works perfectly with the correct Vasi I later downloaded, proving that this was not a bug at all. Thank you for focusing on my issue and sorry for taking you so much trouble!
